apps/dapp-console/app/constants/index.tsx

```diff
@@ -36,35 +36,35 @@ const externalRoutes: Routes = {
   },
   // Documentation items
   ETH_DOCS: {
-    path: '',
+    path: 'https://ethereum.org/developers/docs',
```

Comment:
what happens when this constants file gets bloated?

Reply:
We can separate things out later on if needed. For now there isn't enough to split things up